src/Comment/stories/index.stories.tsx

```diff
@@ -0,0 +1,383 @@
+import React, { ChangeEventHandler, ReactNode } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import { Form } from 'antd'
+import TextArea from 'antd/lib/input/TextArea'
+import Avatar from '../../Avatar'
+import Button from '../../Button'
+import List from '../../List'
+import { default as Comment, CommentProps } from '../index'
+
+export default {
+  title: 'Comment',
+  component: Comment
+} as ComponentMeta<typeof Comment>
+
+const Template: ComponentStory<typeof Comment> = (args: CommentProps) => <Comment {...args} />
+
+export const Primary = Template.bind({})
+
+Primary.args = {
+  label: 'Comment',
+  author: 'Han Solo • A few seconds ago',
+  avatar: <Avatar src="https://joeschmoe.io/api/v1/random" alt="Han Solo" />,
+  content: (
+    <p>
+      We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure),
+      to help people create their product prototypes beautifully and efficiently.
+    </p>
+  )
+}
+
+type CommentEditorPropsType = {
+  avatar: React.ReactElement
+  onCancel?: () => void
+  onChange: ChangeEventHandler<HTMLTextAreaElement>
+  onSubmit: () => void
+  submitting: boolean
+  value: string
+}
+
+type RecursiveCommentsPropsType = CommentProps &
+  CommentEditorPropsType & {
+    key: string
+    parentkey?: string
+    replyToID: string
+    setReplyToID: (value: string | undefined) => void
+  }
+
+type CommentListCommentType = {
+  key: string
+  author: string
+  avatar: React.ReactElement
+  children?: CommentListCommentType[]
+  content: ReactNode
+  datetime: string
+}
+
+type CommentListProps = CommentEditorPropsType & {
+  comments: CommentListCommentType[]
+  commentsLength: number
+  replyToID: string
+  setReplyToID: (value: string | undefined) => void
+}
+
+type ReducerPropsType = {
+  commentsLength: number
+  comments: CommentProps[]
+  submitting: boolean
+  value: string
+  replyToId: string | undefined
+}
+
+type ReducerActionsType = 'handleSubmit' | 'setValue' | 'setIsSubmit' | 'setReplyToID'
+
+const CommentEditor: React.FC<CommentEditorPropsType> = props => (
+  <Comment
+    avatar={props.avatar}
+    content={
+      <>
+        <Form.Item>
+          <TextArea rows={4} onChange={props.onChange} value={props.value} />
+        </Form.Item>
+        <Form.Item>
+          <Button loading={props.submitting} onClick={props.onSubmit} type="primary">
+            Submit
+          </Button>
+          {props.onCancel && (
+            <Button onClick={props.onCancel} type="text">
+              Cancel
+            </Button>
+          )}
+        </Form.Item>
+      </>
+    }
+  />
+)
+
+const RecursiveComments: React.FC<RecursiveCommentsPropsType> = props => {
```

Comment:
These components seem to be a requirement from design and not only for the story. If this is the case, they need to be moved to `index.tsx` instead